keanu-python/keanu/vertex/label.py

```diff
@@ -0,0 +1,54 @@
+from py4j.java_gateway import java_import, JavaObject
+from keanu.base import JavaObjectWrapper
+from keanu.context import KeanuContext
+from typing import List
+
+k = KeanuContext()
+
+java_import(k.jvm_view(), "io.improbable.keanu.vertices.VertexLabel")
+
+
+class _VertexLabel(JavaObjectWrapper):
```

Comment:
`_VertexLabel` takes java objects as argument, which is something you wouldn't expect from a user's perspective

Reply:
Instead of having two classes we could have a class method `from_java_object()`. See top answer here: https://stackoverflow.com/questions/141545/how-to-overload-init-method-based-on-argument-type

Reply:
Discussed offline, this specific approach doesn't work but I'm open to other suggestions so I'll not resolve this conversation yet